### PR TITLE
[no ticket][risk=no] Puppeteer test: Filter Workspaces by Access Level

### PR DESCRIPTION
**Pull request by aweng98**
New test: Workspaces filter by access levels.

### CODE REVIEW

File: e2e/tests/workspace/workspace-filter-display.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import WorkspacesPage from 'app/page/workspaces-page';
+import {signIn} from 'utils/test-utils';
+import WorkspaceCard from 'app/component/workspace-card';
+import ReactSelect from 'app/element/react-select';
+import {config} from 'resources/workbench-config';
+
+describe('Workspaces Filter Select menu tests', () => {
+
+  beforeEach(async () => {
+    await signIn(page, config.collaboratorUsername, config.userPassword);
```
Comment (dolbeew) [inline e2e/tests/workspace/workspace-filter-display.spec.ts:10]: Should the username and password be specified here? I couldn't get logged in when running the test locally (local UI & test API). If I remove username and password params, the test runs and passes.

Comment (aweng98) [reply e2e/tests/workspace/workspace-filter-display.spec.ts:10]: The main user `puppetcitester6` is used in ci job but user doesn't have any workspaces with READER or WRITER role. 
What's the page that you see when login is failing?

Test passed in ci job.
```
PASS  tests/workspace/workspace-filter-display.spec.ts (73.466 s)
  Workspaces Filter Select menu tests
    ✓ Display workspaces by access levels (73031 ms)
```

Comment (dolbeew) [reply e2e/tests/workspace/workspace-filter-display.spec.ts:10]: It fails at the login page:
Screen Shot 2020-09-16 at 9 15 50 AM

Comment (dolbeew) [reply e2e/tests/workspace/workspace-filter-display.spec.ts:10]: ^^ Failure due to wrong password in local .env file

**Review by dolbeew — APPROVED** [review]
LGTM

**Pull request merged**